.eslintrc.cjs

```diff
@@ -221,7 +221,7 @@ module.exports = {
         'vue/max-lines-per-block': [
           'warn',
           {
-            script: 100,
+            script: 150,
```

Comment:
We can reduce it later 🫨

Reply:
Funny how this increases with every PR. 😁 I think we should just remove this rule, while the intention is great, it’s not helpful in practice.

Reply:
haha its a good idea! I think just not right now. I would rather keep pushing it up than removing it though. It keeps us honest